### DISCUSSION TITLE
[Feature Request] Combination of random+custom alias chunks in a single alias

### DISCUSSION THREAD

**Discussion by w3lld0ne** [anonaddy/anonaddy#369] [Feature Requests]
Would be nice to have such feature to auto-generate randomized part along with custom prefix/suffix.
User should be able to choose desired random chunk mode (UUID/etc.), custom text (with included separator if needed), and whether the custom part is a prefix or a suffix (or even in-middle).

The purpose of this is to have custom emails that are still hard to be guessed by someone.
For example, instead of 'skype@subdomain.anonaddy.com' user can get 'skype-a4e1aa94-498d-4cc5-afa3-a3fce6b3bf16@subdomain.anonaddy.com' or 'a4e1aa94-498d-4cc5-afa3-a3fce6b3bf16.skype@site.com' or 'a4e1aa94!skype!a3fce6b3bf16@site.com' (creepy one, I know) etc. This GUID was generated in a separate online service but it would be nice to automate this process.

Update: Another way to do that is to simple provide placeholders such as {UUID}, then let the user to write something like 'skype-{UUID}' for a custom part (not sure whether {} are a valid e-mail chars but if they are, there also should be a possibility to escape them like {{ }}).

**Comment by willbrowningme** [comment]
Thanks for the suggestion, I definitely do plan to add much more customisation and options for alias formats in the near future and will allow something similar to this.

**Comment by Damdrara** [comment]
To pile on to this since it's related, another great option would be hashing, similar to how the random thing works, but basically making it deterministic and thus verifyable.

This site does a great job at that:

https://blame.email

Would be cool to have that option here :)
